content/docs/2.0/scalers/apache-kafka.md

```diff
@@ -29,6 +30,7 @@ triggers:
 - `consumerGroup`: consumer group used for checking the offset on the topic and processing the related lag.
 - `topic`: topic on which processing the offset lag.
 - `lagThreshold` How much the stream is lagging on the current consumer group. Default is 10. Optional.
+- `consumerOffsetReset` the offset reset policy for the consumer. Can be other "earliest" or "latest". Default is "earliest".
```

Comment:
IMHO default should be "latest"

Reply:
or at least that is the one specified in the code

Reply:
I'm going change the one in the code since it is more coherent with the current kafka-scaler behaviour

Reply:
ok